src/lib/data.json

```diff
@@ -41,82 +41,351 @@
+            "budgetPercents": [
+              0.1452, 
+              0.1261, 
+              0.1203, 
+              0.1038, 
+              0.1592, 
+              0.0604, 
+              0.0649, 
+              0.0269, 
+              0.0102, 
+              0.0313, 
+              0.0215, 
+              0.0201, 
+              0.0084, 
+              0.0101, 
+              0.0714, 
+              0.0045, 
+              0.0045, 
+              0.0034, 
+              0.0077],
             "standardDeduction": {
               "single": 12400,
-              "seperate": 12400,
+              "separte": 12400,
               "together": 24800,
               "head": 18650
             }
           },
 
         "2021" : {
-            "percents": [0.1, 0.12, 0.22, 0.24, 0.32, 0.35, 0.37],
-            "single": [0, 9950, 40525, 86375, 164925, 209425, 523600],
-            "seperate": [0, 9950, 40525, 86375, 164925, 209425, 314150],
-            "together": [0, 19900, 81050, 172750, 329850, 418850, 628301],
-            "head": [0, 14200, 54200, 86350, 164900, 209400, 523600],
+            "percents": [
+              0.1, 
+              0.12, 
+              0.22, 
+              0.24, 
+              0.32, 
+              0.35, 
+              0.37],
+            "single": [
+              0, 
+              9950, 
+              40525, 
+              86375, 
+              164925, 
+              209425, 
+              523600],
+            "separte": [
+              0, 
+              9950, 
+              40525, 
+              86375, 
+              164925, 
+              209425, 
+              314150],
+            "together": [
+              0, 
+              19900, 
+              81050, 
+              172750, 
+              329850, 
+              418850, 
+              628301],
+            "head": [
+              0, 
+              14200, 
+              54200, 
+              86350, 
+              164900, 
+              209400, 
+              523600],
             "ss": 142800,
-            "budgetPercents": [0.1374, 0.1178, 0.1111, 0.0998, 0.1987, 0.0576, 0.0696, 0.0256, 0.0195, 0.0505, 0.0218, 0.0125, 0.0078, 0.0087, 0.0387, 0.0041, 0.0036, 0.0018, 0.0131],
+            "budgetPercents": [
+              0.1374, 
+              0.1178, 
+              0.1111, 
+              0.0998, 
+              0.1987, 
+              0.0576, 
+              0.0696, 
+              0.0256, 
+              0.0195, 
+              0.0505, 
+              0.0218, 
+              0.0125, 
+              0.0078, 
+              0.0087, 
+              0.0387, 
+              0.0041, 
+              0.0036, 
+              0.0018, 
+              0.0131],
             "standardDeduction": {
               "single": 12550,
-              "seperate": 12550,
+              "separte": 12550,
```

Comment:
Uh I think it's supposed to be separate, not separte

Reply:
It is Done